examples/sinatra/Gemfile.lock

```diff
@@ -26,7 +26,7 @@ GEM
     omniauth-oauth2 (1.6.0)
       oauth2 (~> 1.1)
       omniauth (~> 1.9)
-    puma (5.6.4)
+    puma (6.3.1)
```

Comment:
I can't see puma being specified as a dependency in the Gemfile or gemspec. How does it show up in the Gemfile.lock?

Reply:
This is inside dir `/examples/sinatra/`.